projects/novo-examples/src/form-controls/value/formatter-value/formatter-value-example.ts

```diff
@@ -21,6 +21,8 @@ export class FormatterValueExample {
     associatedEntity: {
       entity: 'Placement',
     },
+    showEntityIcon: true,
```

Comment:
Do you mean to set the value or type here?

Reply:
Are you referring to the line above or below? The `entityIconClass` was recommended from my team but I'm always open for further suggestions

Reply:
nvm I am dumb